lua/neotest-go/color.lua

```diff
@@ -1,16 +1,136 @@
 local M = {}
 
--- Highlight some parts of the test output
-function M.highlight_output(output)
+local fmt = string.format
+local colors_patterns = require("neotest-go.patterns").colors
+
+local term = os.getenv("COLORTERM")
+local fullcolor = (term == "truecolor") or (term == "24bit")
+
+local function slice(tbl, first, last, step)
+  local sliced = {}
+  for i = first or 1, last or #tbl, step or 1 do
+    sliced[#sliced + 1] = tbl[i]
+  end
+  return sliced
+end
+
+---Escape characters that have a special meaning in patterns
+---@param text string
+---@return string
+local function escape(text)
+  local special_chars = { "(", ")", "%", ".", "+", "-", "*", "[", "?", "^", "$" }
+  for _, char in pairs(special_chars) do
+    text = text:gsub("%" .. char, "%%" .. char)
+  end
+  return text
+end
+
+-- Function:	HEXtoRGB(arg)
+-- Argument:	Hex string value in the form '#cccccc' or 'cccccc'
+-- 						HEX shorthand is supported
+-- Returns:		Three RGB values
+-- 						Red value from 0-255
+-- 						Green value from 0-255
+-- 						Blue value from 0-255
+-- Source:    forum.rainmeter.net/viewtopic.php?t=29419
+local function HEXtoRGB(hexArg)
+  hexArg = hexArg:gsub("#", "")
```

Comment:
Generally this project uses snake case but seems that just this function is using camel case

Reply:
You're right, had just copied it over